src/components/v5/shared/Button/Button.tsx

```diff
@@ -1,106 +1,108 @@
+    const ariaLabelText =
+      typeof ariaLabel === 'string'
+        ? ariaLabel
+        : ariaLabel && formatText(ariaLabel);
 
-  return (
-    <>
-      {loading ? (
-        <SpinnerLoader appearance={{ size: 'medium' }} />
-      ) : (
-        <button
-          className={clsx(
-            'flex items-center justify-center font-medium transition-all duration-normal',
-            {
-              'text-md min-h-[2.5rem] px-4 py-2': size === 'default',
-              'text-md min-h-[2.5rem] px-[0.875rem] py-[0.625rem]':
-                size === 'large',
-              '!rounded-[0.1875rem] capitalize text-4 px-2 py-1':
-                size === 'extraSmall',
-              'text-sm min-h-[2.125rem] px-3 py-2': size === 'small',
-              [styles.primarySolid]: mode === 'primarySolid',
-              [styles.primarySolidFull]: mode === 'primarySolidFull',
-              [styles.primaryOutline]: mode === 'primaryOutline',
-              [styles.primaryOutlineFull]: mode === 'primaryOutlineFull',
-              [styles.secondarySolid]: mode === 'secondarySolid',
-              [styles.secondaryOutline]: mode === 'secondaryOutline',
-              [styles.quinary]: mode === 'quinary',
-              [styles.senary]: mode === 'senary',
-              [styles.quaternary]: mode === 'quaternary',
-              [styles.tertiary]: mode === 'tertiary' || size === 'large',
-              [styles.septenary]: mode === 'septenary',
-              [styles.completed]: mode === 'completed',
-              'pointer-events-none': disabled,
-              'w-full': isFullSize,
-              'bg-blue-400 text-base-white': mode === 'filled',
-              'border border-gray-300 !text-gray-300 !bg-base-white':
-                disabled && isIconRight,
-              'bg-base-white border border-gray-300 text-gray-700 sm:hover:bg-blue-400 sm:hover:text-base-white sm:hover:border-blue-400':
-                mode === 'notFilled',
-              'rounded-full': isFullRounded,
-              'rounded-lg': !isFullRounded,
-            },
-            className,
-          )}
-          disabled={disabled || loading}
-          aria-label={ariaLabelText}
-          aria-busy={loading}
-          title={titleText}
-          type={type}
-          ref={setTriggerRef}
-          {...rest}
-        >
-          <ButtonContent
-            mode={mode}
-            iconName={iconName}
-            iconSize={iconSize}
-            isIconRight={isIconRight}
-            text={text}
-            textValues={textValues}
+    return (
+      <>
+        {loading ? (
+          <SpinnerLoader appearance={{ size: 'medium' }} />
+        ) : (
+          <button
+            className={clsx(
+              'flex items-center justify-center font-medium transition-all duration-normal',
+              {
+                'text-md min-h-[2.5rem] px-4 py-2': size === 'default',
+                'text-md min-h-[2.5rem] px-[0.875rem] py-[0.625rem]':
```

Comment:
`min-h-[2.5rem]` can also be `min-h-10`. No need for an arbitrary value

Reply:
It doesn't work

Reply:
My bad, I think `h-10` exists so that might be why I got confused and assumed it also existed on `min-h`